Composer/packages/client/src/pages/knowledge-base/table-view.tsx

```diff
@@ -456,7 +471,7 @@ const TableView: React.FC<TableViewProps> = (props) => {
                       {
                         key: 'update',
                         iconProps: { iconName: 'Download' },
-                        name: formatMessage('Import new URL and overwrite'),
+                        name: formatMessage('Replace content'),
```

Comment:
![Screen Shot 2021-06-24 at 4 51 58 PM](https://user-images.githubusercontent.com/13004779/123346232-a9cc6280-d50c-11eb-961f-e1275fd22cec.png)
Is the UI in the figma outdated?

Reply:
Hi @srinaath did you find the latest one
![image](https://user-images.githubusercontent.com/49866537/123346637-58af8380-d58b-11eb-9af3-dcb697a8ae87.png)

![image](https://user-images.githubusercontent.com/49866537/123347112-81377d80-d58b-11eb-8612-b44e2a980806.png)

it's a little different than the archived one.
![image](https://user-images.githubusercontent.com/49866537/123349934-8517cf80-d58c-11eb-9871-6cdd0e7f157e.png)